netmanager/README.md

```diff
@@ -1,68 +1,24 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Netmanager Application
+This application is used for monitoring, device registration, location registration. Can be accessed here: https://staging-platform.airqo.net/ and https://platform.airqo.net/
 
-## Available Scripts
+## Local Setup
+npm is our package manager of choice here. Check out setup instructions [here](https://nodejs.org/dist/) if you don't have it installed already. <br/>After that you can run `npm install` to install all the needed dependencies.<br/>
```

Comment:
@Codebmk You may want to put  the package installation (npm install) as its own section/sub-section. Also include the step of setting up the `.env` file (include the link to the google drive file).

Reply:
On it @ja-odur